File: 09 - Remove-Duplicates-LinkedList/09.py
class LinkedList:
    def __init__(self, value):
        self.value = value
        self.next = None


def remove_duplicates_from_linked_list(linked_list):
    current_node = linked_list
    while current_node.next is not None:
        next_node = current_node.next
        if current_node.value == next_node.value:
            current_node.next = next_node.next
        else:
            current_node = next_node

    return linked_list

File: easy/09 - Remove-Duplicates-LinkedList/09.js
class LinkedList {
    constructor(value) {
      this.value = value;
      this.next = null;
    }
  }
  
  function removeDuplicatesFromLinkedList(linkedList) {
      currentNode = linkedList
      while (currentNode.next !== null){
          nextNode = currentNode.next
          if (currentNode.value == nextNode.value) {
              currentNode.next = nextNode.next
          }
          else {
              currentNode = nextNode
          }
      }
      return linkedList
}